src/kirby/components/progress-bar/progress-bar.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
what is this line good for?

as far as i can see you don't have any unused variables.